docs/extensions/engines/spark/rules.md

```diff
@@ -68,17 +68,18 @@ Now, you can enjoy the Kyuubi SQL Extension.
 
```

Comment:
> Kyuubi provides SQL extension out of box. Due to the version compatibility with Apache Spark, currently we only support Apache Spark branch-3.1 (i.e 3.1.1 and 3.1.2).

as we are here, can you also correct the supported version in the L21?

Reply:
good catch !